main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"sync"
+
+	"github.com/rs/zerolog/log"
+	component "github.com/th2-net/th2-box-template-go/component"
+	"github.com/th2-net/th2-common-go/schema/factory"
+	rabbitmq "github.com/th2-net/th2-common-go/schema/modules/mqModule"
+	"github.com/th2-net/th2-common-go/schema/queue/message"
+)
+
+func main() {
+
+	var closingFunctions []func()
```

Comment:
Component should call all closing function in both cases:
* shutdown
* critical problem during initialization or work